working-groups/meta/NOTES.md

```diff
@@ -0,0 +1,36 @@
+# Meta Meeting Notes
+This document contains meeting notes from the Meta working group.
+
+## 2019-02-21:
+**Written by:** [@nikomatsakis][nikomatsakis]
+
+**Purpose of meeting**:
+
+- Clarify the “scope” of the meta working group.
+  - What problems does the meta working group aim to solve?
+  - What problems do we want to avoid creating? (e.g. bureaucracy, administrative overhead)
+- Discuss working group “bootstrap process”
+  - What should each working group define?
+  - What structure should a WG “check-in” have?
+  - Can we produce a “WG Template” that I can then pass around to folks to fill out?
+- What are the most actionable ideas and can we rank those?
+- Which ideas would be better suited for more “discussion meetings”?
+  - Steering meetings? Or should we use those for something else
+
+**Working Group Components**:
+
+- Zulip Stream(s)
+- Zulip User Group (`WG-compiler-foo`)
+- GitHub Team? Someway for people to get notifications
+- GitHub Labels
+- Contact points:
+  - Leads
+  - Compiler team liason — who will present
+  - Maybe others that you can ping on Zulip to ask questions
+  
+- Across team:
+  - Some kind of table summarizing WGs
+
```

Comment:
I know that this was how it was formatted in the paper document, but could you remove the unnecessary newlines between these bullet points?

Reply:
Done.